include/mlir/Dialect/SPIRV/SPIRVOps.td

```diff
@@ -252,7 +255,100 @@ def SPV_ExecutionModeOp : SPV_Op<"ExecutionMode", [ModuleOnly]> {
 
 // -----
 
-def SPV_FMulOp : SPV_Op<"FMul", [NoSideEffect, SameOperandsAndResultType]> {
+def SPV_FAddOp : SPV_ArithmeticOp<"FAdd", SPV_Float> {
```

Comment:
This op should also be marked as `Commutative`. We should add another template parameter to `SPV_ArithmeticOp` to allow additional traits on certain ops. (You can use `!listconcat(traits, [NoSideEffect, SameOperandsAndResultType])` to concatenate them.)

Reply:
Thanks for mention this! Fixed.